client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -90,11 +91,17 @@ class TransferDomainPrecheck extends React.Component {
 	};
 
 	refreshStatus = () => {
-		this.props.refreshStatus( this.statusRefreshed );
+		this.props.refreshStatus( this.statusRefreshed ).then( result => {
+			const isUnlocked = get( result, 'inboundTransferStatus.unlocked' );
+			this.props.recordUnlockedCheckButtonClick( this.props.domain, isUnlocked );
```

Comment:
I'm not sure why calling `recordUnlockedCheckButtonClick` inside of `then()` causes this error:

```Uncaught (in promise) TypeError: Cannot read property 'type' of undefined
    at eval (wpcom-api-middleware.js:82)
    at eval (index.js:12)
    at Object.eval [as recordUnlockedCheckButtonClick] (redux.js:457)
    at eval (transfer-domain-precheck.jsx:125)